src/utils/jsx-entity.ts

```diff
@@ -319,8 +320,9 @@ export interface JSXComponentData extends ComponentData {
     captureDurLblRef: Ref;
     sndToggleRef: Ref;
   };
-  animationMixer?: any;
   mediaLoader?: MediaLoaderParams;
+  mixerAnimatable?: MixerAnimatableParams;
```

Comment:
not sure if we will end up needing an inflator for this or if we need to expose it to JSX. I think it's only valid to be inserted internally as part of a model component, but maybe we will find a use for manually making one in JSX as well... Not sure.